mainframer.sh

```diff
@@ -70,6 +74,27 @@ if [ -z "$REMOTE_COMMAND" ]; then
 	exit 1
 fi
 
+function formatTime {
+    ((h=${1}/3600))
+    ((m=(${1}%3600)/60))
+    ((s=${1}%60))
+
+    #Find out the time format and correct string.
+    if [ "$h" -eq "1" ]; then
+        HOURS_TAG="hour"
+    fi
+    if [ "$m" -eq "1" ]; then
+        MINUTES_TAG="minute"
+    fi
+    if [ "$s" -eq "1" ]; then
+        SECONDS_TAG="second"
+    fi
+
+    (( h > 0 )) && printf "%d $HOURS_TAG " ${h}
+    (( m > 0 )) && printf "%d $MINUTES_TAG " ${m}
+    (( s > 0 )) && printf "%d $SECONDS_TAG \n" ${s}
+}
```

Comment:
Ok @ming13, got you, changed my code to your implementation mostly 👍  Thanks again for reviewing and helping me to implement this